### ISSUE TITLE
Adding a pdbConverter

### ISSUE THREAD

**Issue by zhu0619** [polaris-hub/polaris#172]
### Discussed in https://github.com/polaris-hub/polaris/discussions/104

<div type='discussions-op-text'>

<sup>Originally posted by **dessygil** May 21, 2024</sup>
Hi all,

I'm currently trying to add the dataset suggested in a previous [discussion post](https://github.com/polaris-hub/polaris/discussions/87) called [PDE10A](https://pubmed.ncbi.nlm.nih.gov/36153472/). In this dataset, there are various files representing the ligand, pocket and protein. The ligand is within a .sdf file, whereas the pocket and protein are represented in a .pdb or .mol2 file. Polaris already has the function to convert a .sdf file to be used but I would like to add a .pdb converter to add the functionality to Polaris and to use for the specific dataset I'm trying to add.

In short, I'm writing this post to discuss the best strategy for adding the pdb converter.</div>
